### PR TITLE
cmac/BFD-690 patient resource carin mappings

### PR DESCRIPTION
**Pull request by mackec1**
### Change Details

No real changes in CARIN mappings; removed HICN related code from Provider and Transformer classes.

- apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java
- apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2Test.java

### Acceptance Validation

Verify CARIN mapping as needed.

### Feedback Requested

Code style and/or logic errors.

### External References

- [BFD-690](https://jira.cms.gov/browse/BFD-690)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [X] new data stored or transmitted

V2 CARIN mappings.

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java [hunk 1]
```diff
@@ -582,11 +582,9 @@ private String partDFieldByMonth(CcwCodebookVariable month) {
 
     if (requestHeader.isMBIinIncludeIdentifiers()) {
       joinsClause += "left join fetch b.medicareBeneficiaryIdHistories ";
-      passDistinctThrough = true;
```
Comment (Dranith) [inline apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java:585]: Is the removal of these intended?

Comment (mackec1) [reply apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/PatientResourceProvider.java:585]: yes; distinctPassThru true cause the db engine to perform the distinct (unique) processing while setting to false causes the JDBC driver to perform the distinct. For a simple parent-child relationship (which this is) it is more performance to have the JDBC driver handle things.

**Review by Dranith — APPROVED** [review]
Looks good.

**Review by cbrunefearless — APPROVED** [review]
lgtm

**Pull request merged**